### PR TITLE
Docker development environment for simple testing

### PR DESCRIPTION
**Pull request by being-peace**
I added a docker environment, such that you can simply set up a test environment on Linux or Windows Docker with

  cd example
  docker-compose up -d

See also: https://sourceforge.net/p/mrbs/patches/111/

### CODE REVIEW

File: docker_app/HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+However, when configuration of php and database is change, you have to reset the containers.
+
+### Reset containers
+
+* Stop Docker containers:
+  ~~~
+  docker-compose down
+  ~~~
+* Delete persited volumes:
```
Comment (pa-jberanek) [inline docker_app/HOWTO.md:51]: A small typo here, pesited -> persisted

**Review by pa-jberanek — CHANGES_REQUESTED** [review]
Thanks for this - looks good apart from the typo I spotted. For interest I took your idea and ran with it to generate Kubernetes config, an alternative to Docker Compose.

**Comment by jberanek** [comment]
Apologies, this review was meant to be from my personal GitHub account...

**Comment by being-peace** [comment]
> Thanks for this - looks good apart from the typo I spotted. 
@jberanek  Thanks!

> For interest I took your idea and ran with it to generate Kubernetes config, an alternative to Docker Compose.
I must admit that I am rather new to this container thing. The docker files gives me a super easy development environment on Windows and Linux. I don´t know, what is the advantage of a Kubernetes config.

Is there anything you could commit here?

**Comment by jberanek** [comment]
So, this is where I'm not convinced - Kubernetes would arguably be a better way to deploy MRBS in production, but realistically how many MRBS installers would have a Kubernetes cluster available, or would build one to host MRBS...?

**Comment by jberanek** [comment]
The in-progress k8s config I built https://github.com/jberanek/mrbs-k8s

**Comment by being-peace** [comment]
Ok, thanks. Could you describe, what you gain from this? Do I get it right: if you have a k8s Environment you can easily start mrbs for development purposes there. For productive systems there is manual work to do because of the fixed passwords…?

**Pull request merged**